doc/pages/MAINPAGE.md

```diff
@@ -4,4 +4,95 @@ To get started on the project, all the instructions are linked into the README:
 @subpage readme
 
 To get started on Olivier's thesis, here is the documentation related to our implementation of the metaheuristic:
-@subpage learning_algorithms
\ No newline at end of file
+@subpage learning_algorithms
+
+# Fast Preference Learning
```

Comment:
Markdown wise I would recommend having only one title at `#` level, I also find it a bit more esthetic to have a big title and sub titles for the rest. However it's up to you for the change ;)

Reply:
yes defintely